src/pages/starter-kit/checkout/connect.md

```diff
@@ -18,17 +18,19 @@ To use the Adobe Commerce HTTP Client, update `COMMERCE_BASE_URL=<commerce_insta
 
 Depending on your Adobe Commerce setup, there are two options to authenticate and communicate with App Builder:
 
-- [Configure Adobe Identity Management Service (IMS)](#configure-adobe-identity-management-service-ims)
-
 - [Configure Commerce Integration](#create-a-commerce-integration)
 
-If a Commerce integration is detected, it has precedence over IMS authentication. However, if neither option is detected or configured, than client instantiation will fail.
+- [Configure Adobe Identity Management Service (IMS)](#app-builder-authentication-ims)
+
+If a Commerce integration is detected, it has precedence over IMS authentication. However, if neither option is detected or configured, then client instantiation will fail.
+
+### App Builder authentication (IMS)
 
-### Configure Adobe Identity Management Service (IMS)
+Use the following steps to create OAuth credentials for App Builder authentication:
 
-IMS authentication requires the following setup:
+<InlineAlert variant="info" slots="text"/>
```

Comment:
I'd say the AdminSDK info should be part of `Getting started` instead of in `Connect` because this is related to IMS but more like a consequence of willing to use AdminSDK